.github/workflows/catalog.yml

```diff
@@ -1,4 +1,9 @@
-on: [push]
+on:
+  push:
+    paths:
+      - '.github/workflows/catalog.yml'
+      - 'catalog/**'
+      - 'shared/**'
```

Comment:
@nl0: does anything besides the catalog depend on this?

Reply:
no, i don't think so